seguridad/views.py

```
@@ -7,25 +7,22 @@
-                    data['error'] = 'Login Fallido!, usuario no esta habilitado'
+        cuenta = str(request.POST.get('')).strip()
+        user = authenticate(username=cuenta,password=request.POST['password'])
+        if user is not None:
+            if user.is_active:
+                login(request, user)
+                data['resp'] = True
+                data['user'] = user.username
+                return JsonResponse(data, status=200)
             else:
-                data['error'] = 'Login Fallido!, credenciales incorrectas.'
+                data['error'] = 'Login Fallido!, usuario no esta habilitado'
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)